poonam/JS/Assignment13_js/que1.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Date</title>
+    </head>
+    <body>
+        <script>
+            let date = new Date("Feb 21, 2012, 03:12:00");
```

Comment:
We need to Convert Date to Local Format Like this
Add only that new date part
```suggestion
            let date = new Date("Feb 21, 2012, 03:12:00");
            date = date.toLocaleString('en-US', { month: 'short', day: 'numeric', year: 'numeric', 'hour': 'numeric', minute: 'minute', hour12: true });
```